sonaranalyzer-dotnet/rspec/cs/S4564_c#.html

```diff
@@ -0,0 +1,36 @@
+<p>ASP.NET is coming with a feature looking at HTTP requests to prevent potentially dangerous content to perform a cross-site scripting (XSS) attack.
```

Comment:
Suggested wording change: _ASP.Net has a feature to validate HTTP requests..._

Reply:
@agigleux Could you update the RSPEC?

Reply:
Done.